core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -872,6 +882,36 @@ protected void doRun() throws Exception {
     }
 
     /**
+     * Get the Jenkins {@link StartupType startup type}.   
+     * @return The Jenkins {@link StartupType startup type}.
+     * @since FIXME
```

Comment:
Fair point. Thanks.  I added the `@since` earlier and then never thought to remove it once I added the `@Restricted`.